_gsocblogs/2022/blog_IRIS-HEP_KushKothari.md

```diff
@@ -2,7 +2,7 @@
 project: IRIS-HEP
 title: "Uproot + Dask: Blog 1"
 author: Kush Kothari
-date: 25.07.2022
+date: 09.09.2022
```

Comment:
I'm not sure if this is supposed to be a single page, merging the first half and the second, but if it is, then then "Blog 1" name should probably just be "Blog".